diff -Naur ocamlnet-3.6.1.orig/Makefile.rules ocamlnet-3.6.1/Makefile.rules
--- ocamlnet-3.6.1.orig/Makefile.rules	2012-12-31 20:58:00.000000000 +0300
+++ ocamlnet-3.6.1/Makefile.rules	2012-12-31 20:58:11.000000000 +0300
@@ -148,7 +148,7 @@
 .PHONY: install-findlib
 install-findlib: META
 	files=`$(COLLECT_FILES) *.mli *.cmi *.cma *.cmxa *.a dll* META $(INSTALL_EXTRA)` && \
-	$(OCAMLFIND) install $(PKGNAME) $$files
+	$(OCAMLFIND) install $(OCAMLFIND_INSTFLAGS) $(PKGNAME) $$files
 
 .PHONY: uninstall-findlib
 uninstall-findlib:
