--- src/equeue-gtk2/uq_gtk.ml.orig	2007-06-11 19:35:10.000000000 +0100
+++ src/equeue-gtk2/uq_gtk.ml	2007-06-11 20:07:54.000000000 +0100
@@ -3,6 +3,11 @@
 open Equeue
 open Unixqueue
 
+(* Ugly hack required to compile under OCaml 3.10.  This unmarshals
+ * `IN.  IN is a reserved word for Camlp4.Macro.
+ *)
+let __in = (Marshal.from_string "\132\149\166\190\000\000\000\003\000\000\000\000\000\000\000\000\000\000\000\000\001?\229" 0 : GMain.Io.condition)
+
 IFDEF GTK1 THEN
   type event_id = GMain.Io.event_source
 ELSE
@@ -147,7 +152,7 @@
 	       gtk_event_source_hup = None; } in
 	 ( match g.gtk_event_source_in with
 	       None when i ->
-		 g.gtk_event_source_in <- Some(mk_handler `IN [fd] [] []);
+		 g.gtk_event_source_in <- Some(mk_handler __in [fd] [] []);
 	     | Some s when not i ->
 		 dest_handler s;
 		 g.gtk_event_source_in <- None
