diff -Naur ocamlnet-3.6.1/src/netsys/configtests/posix_shm.c ocamlnet-3.6.1.new/src/netsys/configtests/posix_shm.c
--- ocamlnet-3.6.1/src/netsys/configtests/posix_shm.c	2012-11-07 03:01:37.000000000 +0300
+++ ocamlnet-3.6.1.new/src/netsys/configtests/posix_shm.c	2012-12-31 20:57:03.000000000 +0300
@@ -13,7 +13,7 @@
 
     ret = shm_open("/thisFileDoesNotExist", O_RDONLY, 0);
     if (ret == -1 && errno == ENOENT) {
-	return Val_int(0);  /* success */
+	return Val_int(1);  /* success */
     };
-    return Val_int(1);
+    return Val_int(0);
 }
