diff -Naur ocamlnet-2.2.4/src/netsys/configtests/posix_shm.c ocamlnet-2.2.4-fix-shm-test/src/netsys/configtests/posix_shm.c
--- ocamlnet-2.2.4/src/netsys/configtests/posix_shm.c	2007-01-07 17:01:00.000000000 +0100
+++ ocamlnet-2.2.4-fix-shm-test/src/netsys/configtests/posix_shm.c	2007-02-21 22:45:00.000000000 +0100
@@ -11,7 +11,7 @@
 
     ret = shm_open("/thisFileDoesNotExist", O_RDONLY, 0);
     if (ret == -1 && errno == ENOENT) {
-	return Val_int(0);
+	    return Val_int(1);
     };
-    return Val_int(1);
+    return Val_int(0);
 }
